INCWaveMountainsView/Classes/INCWaveMountainLayer.h

```diff
@@ -21,19 +23,37 @@
 
 @interface INCWaveMountainLayer : CAShapeLayer
 
-@property(nonatomic,strong,nonnull)CAShapeLayer *shapeLayerMountain;
-@property(nonatomic,assign)float mountainPercent;
-@property(atomic,strong,nullable)NSNumber *mountainPointId;
-@property(nonatomic,strong,nonnull)CAShapeLayer *maskMountain;
-@property(nonatomic,strong,nonnull)CAGradientLayer *gradientMountain;
 
-@property(nonatomic,assign)BOOL unblockMontainsForMissingPercents;
-@property(nonatomic,weak,nullable)id<INCWavwMountainLayerDelegate> delegate;
+@property(nonatomic,strong,nonnull)CAShapeLayer *shapeLayerMountain;//This property define the shape of the mountain
+@property(nonatomic,assign)float mountainPercent;//This define the hight of the mountain, the value has to be between 0 to 1 or NO_PERCENT_VALUE. A lower value will be considered 0 and higher value will be considered 1.
+@property(atomic,strong,nullable)NSNumber *mountainId;//This is the id of the mountain
+@property(nonatomic,strong,nonnull)CAShapeLayer *maskMountain;//Mask layer
+@property(nonatomic,strong,nonnull)CAGradientLayer *gradientMountain;//Gradient colour
+
+@property(nonatomic,assign)BOOL unblockMontainsForMissingPercents;//If yes the time can be set up to unblock a mountain automatically. If after unblockTimeInterval time the layer doesn't new values the mountain is reset to 0.
```

Comment:
Those comments create an auto documentation in XCode

https://www.raywenderlich.com/66395/documenting-in-xcode-with-headerdoc-tutorial